RELEASING.adoc

```diff
@@ -7,6 +7,8 @@ In order to simplify that process, here is a list of prerequisites to be perform
 
 2. For any non-GA modules (e.g., Spanner), also comment them out from the `pom.xml` file.
 
-3. Manually change the value of `TRACKING_HEADER_PROJECT_VERSION` in link:spring-cloud-gcp-core/src/main/java/org/springframework/cloud/gcp/core/UsageTrackingHeaderProvider.java[`UsageTrackingHeaderProvider`] to the version being released.
+3. Manually change the value of `TRACKING_HEADER_PROJECT_VERSION` in link:spring-cloud-gcp-core/src/main/java/org/springframework/cloud/gcp/core/UsageTrackingHeaderProvider.java[`UsageTrackingHeaderProvider`] to the version being released from the interim `SNAPSHOT` value.
```

Comment:
Well, actually it is called `BUILD-SNAPSHOT` do not mislead a consumer of this document.

Reply:
Changed. 

I thought it would be clear which value I am referring to, since there are also numbers in front of the "BUILD-" that we cannot mention here.